Files/Dialogs/FilesystemOperationDialog.xaml

```diff
@@ -0,0 +1,63 @@
+﻿<ContentDialog
+    x:Class="Files.Dialogs.FilesystemOperationDialog"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:helpers="using:Files.Helpers"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    RequestedTheme="{x:Bind helpers:ThemeHelper.RootTheme}"
+    Title="{x:Bind ViewModel.Title, Mode=OneWay}"
+    PrimaryButtonText="{x:Bind ViewModel.PrimaryButtonText, Mode=OneWay}"
+    SecondaryButtonText="{x:Bind ViewModel.SecondaryButtonText, Mode=OneWay}"
+    CloseButtonText="{x:Bind ViewModel.CloseButtonText, Mode=OneWay}">
+
+    <Border>
+        <Grid>
+            <StackPanel Spacing="10">
+                <TextBlock Text="{x:Bind ViewModel.Subtitle, Mode=OneWay}"/>
+
+                <CheckBox
+                    x:Name="chkPermanentlyDelete"
+                    x:Uid="ConfirmDeleteDialogPermanentlyDeleteCheckBox"
+                    x:Load="{x:Bind ViewModel.PermanentlyDeleteLoad, Mode=OneWay}"
+                    BorderThickness="0"
+                    Content="Permanently delete"
+                    IsEnabled="{x:Bind ViewModel.PermanentlyDeleteEnabled, Mode=OneWay}"
+                    IsChecked="{x:Bind ViewModel.PermanentlyDelete, Mode=TwoWay}" />
+
+                <Button Padding="0"
+                        Background="Transparent"
+                        Command="{x:Bind ViewModel.ExpandDetailsCommand, Mode=OneWay}"
+                        CommandParameter="{x:Bind ViewModel}">
+                    <Button.Content>
+                        <StackPanel Orientation="Horizontal" Spacing="5">
+                            <FontIcon
+                                x:Name="Expand"
+                                x:Load="{x:Bind ViewModel.ChevronDownLoad, Mode=OneWay}"
+                                FontSize="14"
+                                Glyph="&#xE70D;" />
+                            <FontIcon
+                                x:Name="Collapse"
+                                x:Load="{x:Bind ViewModel.ChevronUpLoad, Mode=OneWay}"
+                                FontSize="14"
+                                Glyph="&#xE70E;" />
+                            <TextBlock Text="Details" FontWeight="Bold"/>
+                        </StackPanel>
+                    </Button.Content>
+                </Button>
```

Comment:
Do you think we can use the WCT expander here?

Reply:
nah

Reply:
@d2dyno1 Don't you think this seems like extra overhead when we can use an existing control?

Reply:
Does x:Load work with WCT Expander? @yaichenbaum 
It does with the current solution.

Reply:
There isn't any reason why it shouldn't...

Reply:
I've just tested out the wct expander and it doesn't fit the UI

Reply:
@yaichenbaum 
![image](https://user-images.githubusercontent.com/53011783/113346966-cf463780-9334-11eb-9aa6-f346d6ea909d.png)